functions/inputs/from_test.go

```diff
@@ -28,6 +30,14 @@ func fluxTime(t int64) flux.Time {
 	}
 }
 
+func makeFilterFn(exprs ...semantic.Expression) *semantic.FunctionExpression {
+	body := semantic.ExprsToConjunction(exprs...)
+	return &semantic.FunctionExpression{
+		Params: []*semantic.FunctionParam{{Key: &semantic.Identifier{Name: "r"}}},
```

Comment:
If you made "r" an input to this function and tested with different "r" values, I think you'd find find a bug in `isPushableExpr`. This isn't necessary if you fix `isPushableExpr`.